frontend/src/app/Main.tsx

```diff
@@ -44,7 +44,37 @@ const Main: React.FC<ViewProps> = () => {
       }}
       renderDrawerContent={() => {
         return (
-          <Text style={SpacingStyle.pad(drawerInsets, 16)}>Drawer content</Text>
+          <View style={SpacingStyle.pad(drawerInsets, 16)}>
+            <TouchableOpacity onPress={()=>{}}>
+                <View style={styles.drawerItem}>
+                  <MaterialIcons name="accessible" size={24} color="black" /> 
+                  <Text style={styles.drawerItemText}>ADA Ride Request</Text>
+                </View>
+            </TouchableOpacity>
+            
+            <TouchableOpacity onPress={()=>{}}>
+                <View style={styles.drawerItem}>
+                  <MaterialIcons name="error-outline" size={24} color="black" /> 
+                  <Text style={styles.drawerItemText}>Upcoming Outages</Text>
+                </View>
+            </TouchableOpacity>
+
+            <TouchableOpacity onPress={()=>{}}>
+                <View style={styles.drawerItem}>
+                  <MaterialIcons name="settings" size={24} color="black" /> 
+                  <Text style={styles.drawerItemText}>Settings</Text>
+                </View>
+            </TouchableOpacity>
+  
+            <TouchableOpacity onPress={()=>{}}>
+                <View style={styles.drawerItem}>
+                  <MaterialIcons name="bug-report" size={24} color="black" /> 
+                  <Text style={styles.drawerItemText}>Bug Report</Text>
+                </View>
+            </TouchableOpacity>
+  
+            <Text>Version 1.0.0</Text>
```

Comment:
I think Expo's build config equivalent has a variable we can use here

Reply:
I added this, but it's returning a weird number. I think it may be something on Expo's end or our configuration.

Reply:
Nvm, got it